### PR TITLE
Added the swift at root and object torrent scripts to CI

### PR DESCRIPTION
**Pull request by TejasC88**
Related PRs : 
https://github.com/red-hat-storage/ceph-qe-scripts/pull/619
https://github.com/red-hat-storage/ceph-qe-scripts/pull/622

Integration of the 2 above mentioned PRs to cephCI
Polarion: 
CEPH-83572699
CEPH-83574614

### CODE REVIEW

File: suites/reef/rgw/tier-2_rgw_regression_test.yaml [hunk 1]
```diff
@@ -203,6 +203,16 @@ tests:
...
 
+  - test:
+      name: Test the swift URL with '/'
+      desc: Test the swift URL with '/'
+      polarion-id: CEPH-83572699
+      module: sanity_rgw.py
+      config:
+        script-name: test_swift_basic_ops.py
+        config-file-name: test_swift_at_root.yaml
+        timeout: 300
```
Comment (ckulal) [inline suites/reef/rgw/tier-2_rgw_regression_test.yaml:214]: Please remove timeout

Comment (TejasC88) [reply suites/reef/rgw/tier-2_rgw_regression_test.yaml:214]: Done

**Comment by TejasC88** [comment]
Keeping the DNM flag , as its causing a failure detailed here : https://github.com/red-hat-storage/ceph-qe-scripts/pull/623

**Comment by ckulal** [comment]
@tejas, Please update the description and appropriate label.
and since ceph-qe-scrpt PR https://github.com/red-hat-storage/ceph-qe-scripts/pull/623 is merged,
Can you please provide one pass log

**Comment by TejasC88** [comment]
> @tejas, Please update the description and appropriate label. and since ceph-qe-scrpt PR [red-hat-storage/ceph-qe-scripts#623](https://github.com/red-hat-storage/ceph-qe-scripts/pull/623) is merged, Can you please provide one pass log

Removed the DNM flag , pass log : http://magna002.ceph.redhat.com/cephci-jenkins/tchandra/cephci-run-LJF6I5/
pls review

**Comment by ckulal** [comment]
> > @tejas, Please update the description and appropriate label. and since ceph-qe-scrpt PR [red-hat-storage/ceph-qe-scripts#623](https://github.com/red-hat-storage/ceph-qe-scripts/pull/623) is merged, Can you please provide one pass log
> 
> Removed the DNM flag , pass log : http://magna002.ceph.redhat.com/cephci-jenkins/tchandra/cephci-run-LJF6I5/ pls review

@TejasC88 , if failure here is a known issue could you please include comment as known issue

**Comment by TejasC88** [comment]
> > > @tejas, Please update the description and appropriate label. and since ceph-qe-scrpt PR [red-hat-storage/ceph-qe-scripts#623](https://github.com/red-hat-storage/ceph-qe-scripts/pull/623) is merged, Can you please provide one pass log
> > 
> > 
> > Removed the DNM flag , pass log : http://magna002.ceph.redhat.com/cephci-jenkins/tchandra/cephci-run-LJF6I5/ pls review
> 
> @TejasC88 , if failure here is a known issue could you please include comment as known issue

@ckulal the known issue for s3cmd failure has already been marked in a previous PR , so we are good there

**Comment by ckulal** [comment]
> > > > @tejas, Please update the description and appropriate label. and since ceph-qe-scrpt PR [red-hat-storage/ceph-qe-scripts#623](https://github.com/red-hat-storage/ceph-qe-scripts/pull/623) is merged, Can you please provide one pass log
> > > 
> > > 
> > > Removed the DNM flag , pass log : http://magna002.ceph.redhat.com/cephci-jenkins/tchandra/cephci-run-LJF6I5/ pls review
> > 
> > 
> > @TejasC88 , if failure here is a known issue could you please include comment as known issue
> 
> @ckulal the known issue for s3cmd rate limit failure has already been marked in a previous PR , so we are good there

Okay, looks good

**Review by ckulal — COMMENTED** [review]
LGTM

**Review by mkasturi18 — APPROVED** [review]

**Pull request merged**